### ISSUE TITLE
2023.9: forecast-related changes

### ISSUE THREAD

**Issue by ildar170975** [Yevgenium/weather-chart-card#130]
2023.9 introduced changes related to forecast data.
Seems that the card need to be updated to support these changes.

**Comment by pimisen** [comment]
@Yevgenium yes, if i understood correctly we now need to use the weather.get_forecast function: [Link](https://www.home-assistant.io/blog/2023/09/06/release-20239/#weather-forecast-service). As I am not a developer, I can't do the changes on my own and check them in :-(

**Comment by slecram** [comment]
@Yevgenium could you fix this issue as soon as possible, please?

**Comment by slecram** [comment]
@Yevgenium ?

**Comment by pimisen** [comment]
So sad, that this great thing obviously is dead :-(

**Comment by override80** [comment]
You can refer to this to make it work again, as a workaround:

https://github.com/hg1337/homeassistant-dwd/blob/f47840bfade5ed21781843542674f7ccb6be0ba3/questions_and_answers.md#im-using-a-third-party-weather-card-that-doesnt-support-the-new-forecast-mechanism-can-i-continue-using-it
